include/sympack/Environment.hpp

```diff
@@ -80,8 +98,12 @@ namespace symPACK{
  **********************************************************************/
 
 namespace symPACK{
-
-
+  template <typename Map>
+  inline void increment_counter(Map &map, std::string const &op) {
+    if (symPACK::gpu_verbose) {
+        map[op]+=1;
+    }
```

Comment:
@Hooninator This is now failing to compile in non-CUDA mode, because `symPACK::gpu_verbose` and `gpu_ops/cuda_ops` require CUDA_MODE

I suggest:

```suggestion
  #ifdef CUDA_MODE
    if (symPACK::gpu_verbose) {
        map[op]+=1;
    }
  #endif
```
And also hoisting `cpu_ops, gpu_ops` definitions out of `CUDA_MODE` to avoid compile errors (although they will be unused in non-CUDA mode)

Reply:
Whoops, should have caught this. Thank you.